unsafe/src/main/java/org/apache/spark/unsafe/bitset/BitSetMethods.java

```diff
@@ -71,7 +72,13 @@ public static boolean isSet(Object baseObject, long baseOffset, int index) {
    * Returns {@code true} if any bit is set.
    */
   public static boolean anySet(Object baseObject, long baseOffset, long bitSetWidthInBytes) {
-    for (int i = 0; i <= bitSetWidthInBytes; i++) {
+    long widthInLong = bitSetWidthInBytes / SIZE_OF_LONG;
+    for (long i = 0; i <= widthInLong; i++) {
```

Comment:
actually we should use int instead of long for i. the reason is that JIT would inject a safepoint for loops over longs which prevents loop unrolling and incurs an extra check for every iteration